agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -0,0 +1,179 @@
+#script to log and readout pfeiffer TPG 366 gauge contoller
+#via Ethernet connection
+#Zhilei Xu, Tanay Bhandarkar
+
+import socket
+import numpy as np
+from ocs import ocs_agent, site_config, client_t
+from ocs.ocs_twisted import TimeoutLock
+import time
+
+BUFF_SIZE = 128
+ENQ = '\x05'
+IP_ADDRESS = '10.10.10.20'
+
+class pfeiffer:
+    """
+    CLASS to control and retrieve data from the pfeiffer tpg366 
+    pressure gauage controller
+    ip_address: ip address of the deivce
+    port: 8000 (fixed for the device)
+    
+    Attributes:
+       read_pressure reads the pressure from one channel (given as an argument)
+       read_pressure_all reads pressrue from the six channels
+       close closes the socket
+    
+    Todo:
+        -Generalize IP address inputs (input IP  via command line?)
+    """
+    def __init__(self, ip_address=IP_ADDRESS, port=8000, timeout=10, f_sample = 2.5):
```

Comment:
You should remove the default argument of `IP_ADDRESS` for `ip_address` and expect the IP to be passed in when a pfeiffer object is instantiated in the Agent. (You'll see later, you have an `ip_address` argument for the Agent, but don't actually use it to set the IP address of the device, relying on this default, hard coded value.)